frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -45,11 +45,11 @@ export default class DebouncedFrame extends React.Component {
       return;
     }
     if (
-      this.props.width !== nextProps.width ||
-      this.props.height !== nextProps.height
+      this.state.width !== nextProps.width ||
+      this.state.height !== nextProps.height
```

Comment:
The problem, in this case, is that we only compared new and previous props, but while the `DebouncedFrame` is disabled, no state will be updated which means while the map visualization is up this component would have `enabled == true` so no state is updated and by the time we change the visualazation back to the table, `this.props.width,height` would already be equal to `nextProps.width,height`.

This change uses similar paradigm to using `useState` + updating the state when `props` changed, but in this case, I'm always making sure that `state` and `nextProps` would be in sync when `enabled == true`.